### ISSUE TITLE
java.lang.NumberFormatException: For input string: "h7v0l43"

### ISSUE THREAD

**Issue by edrevo** [sbt/sbt#5218]
## steps
sbt version: 1.3.3 (this bug was introduced in 1.3.0)

## problem
```
sbt:baikal-services> last common/update
[debug] "not up to date. inChanged = true, force = true
[debug] Updating common...
[error] java.lang.NumberFormatException: For input string: "h7v0l43"
[error]         at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
[error]         at java.lang.Integer.parseInt(Integer.java:580)
[error]         at java.math.BigInteger.<init>(BigInteger.java:479)
[error]         at lmcoursier.internal.shaded.coursier.cache.CacheChecksum$$anonfun$findChecksum$1.applyOrElse(CacheChecksum.scala:22)
[error]         at lmcoursier.internal.shaded.coursier.cache.CacheChecksum$$anonfun$findChecksum$1.applyOrElse(CacheChecksum.scala:20)
[error]         at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:148)
[error]         at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:135)
[error]         at scala.collection.AbstractTraversable.collectFirst(Traversable.scala:108)
[error]         at lmcoursier.internal.shaded.coursier.cache.CacheChecksum$.findChecksum(CacheChecksum.scala:20)
[error]         at lmcoursier.internal.shaded.coursier.cache.CacheChecksum$.$anonfun$parseChecksumAlternative$2(CacheChecksum.scala:31)
[error]         at scala.Option.orElse(Option.scala:447)
[error]         at lmcoursier.internal.shaded.coursier.cache.CacheChecksum$.parseChecksumAlternative(CacheChecksum.scala:30)
[error]         at lmcoursier.internal.shaded.coursier.cache.CacheChecksum$.$anonfun$parseRawChecksum$1(CacheChecksum.scala:58)
[error]         at scala.Option.orElse(Option.scala:447)
[error]         at lmcoursier.internal.shaded.coursier.cache.CacheChecksum$.parseRawChecksum(CacheChecksum.scala:58)     [error]         at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$validateChecksum$4(FileCache.scala:723)  [error]         at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[error]         at scala.util.Success.$anonfun$map$1(Try.scala:255)
[error]         at scala.util.Success.map(Try.scala:213)
[error]         at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[error]         at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[error]         at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[error]         at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[error]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[error]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[error]         at java.lang.Thread.run(Thread.java:748)
```

## expectation
It should update correctly.

## notes
I don't know how to minimally reproduce this issue. If there is any command I can run to give you more insight on what is going on inside of SBT or Coursier, I'm happy to run them.

**Comment by eed3si9n** [comment]
@edrevo Thanks for the report. Looking at the stacktrace it looks like some internal of Coursier, so could you also report to coursier/coursier please?

**Comment by edrevo** [comment]
Fixed in SBT 1.3.13
